habushu-maven-plugin/pom.xml

```diff
@@ -191,6 +209,9 @@
                             cucumber.junit-platform.naming-strategy=long
                         </configurationParameters>
                     </properties>
+                    <excludes>
+                        <exclude>**/ContainerizeDependenciesMojoTestCase.java</exclude>
```

Comment:
Q: Why is this necessary for this class, but not the extremely similar `MojoTestCaseWrapper` in fermenter?

Reply:
per Ryan's comment, it's due to the naming of the class. this has been fixed